recipes/ablog/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "ablog" %}
+{% set version = "0.8.4" %}
+{% set sha256 = "abb943a93078f5a060cee8f5dbfc44bd65bd1bef448c099e072265fab9ee57a9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
I think you may need `entry_points` https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

Reply:
I know for our recipes we use `python setup.py install --offline --no-git --single-version-externally-managed --record record.txt`, I am unsure if that information is helpful tho!

Reply:
@CJ-Wright setup.py already has the entry points. Do I need to list them again?

@nabobalis I'm just using the default `script:` from the example.  Is there a reason why we need something custom?

Reply:
Honestly, I'm not sure. It just what happens to be the method we use for our python packages and I have not spent any time playing around with the CI on conda to know for sure.